ReText/window.py

```diff
@@ -353,6 +353,18 @@ def __init__(self, parent=None):
 		self.fileSystemWatcher = QFileSystemWatcher()
 		self.fileSystemWatcher.fileChanged.connect(self.fileChanged)
 
+		# Restore last opened files
+		if globalSettings.openLastFilesOnStartup:
+			files = readListFromSettings("lastFileList")
+			for file in files:
+				self.openFileWrapper(file)
+
+			# Trigger the tab of last opened file
+			lastTabIndex = readFromSettings(
+				"lastTabIndex", int, default=self.tabWidget.currentIndex())
```

Comment:
The `currentIndex` is (almost) always zero, so you can add `lastTabIndex` to the list of settings and use `globalSettings.lastTabIndex` instead.